pkgs/tools/networking/network-manager/fortisslvpn/default.nix

```diff
@@ -27,10 +27,14 @@ in stdenv.mkDerivation {
   configureFlags = [
     "--without-libnm-glib"
     "--with-gnome=${if withGnome then "yes" else "no"}"
-    "--localstatedir=/tmp"
+    "--localstatedir=/var"
     "--enable-absolute-paths"
   ];
 
+  # the installer only create an empty directory in localstatedir, so
+  # we can drop it
+  installFlags = [ "localstatedir=." ];
```

Comment:
can $TMPDIR be used?

Reply:
why not, but honestly it does not improve anything.